_posts/2021-xx-xx-release-1.10.3.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title:  "Apache Flink 1.10.3 Released"
+date:   2021-xx-xx 00:00:00
+categories: news
+authors:
+- xtsong:
+  name: "Xintong Song"
+---
+
+The Apache Flink community released the third bugfix version of the Apache Flink 1.10 series.
+
+This release includes 36 fixes and minor improvements for Flink 1.10.2. The list below includes a detailed list of all fixes and improvements.
```

Comment:
I believe we still could not upload the artifacts to PyPI for the time being because of the same reason as 1.12.1(the space limit in PyPI). Could you update the release note for this?

Reply:
Do we need to provide wheel packages for 1.10.x? According to the release process, we start providing wheel packages since 1.11.
https://cwiki.apache.org/confluence/display/FLINK/Creating+a+Flink+Release

Reply:
We will not provide the wheel packages for 1.10.x. There is only one package needed to upload for 1.10.x. However, I'm afraid we still could not make it for the space limit reason.

Reply:
True. I'll add a notice.